src/main/java/org/cloudburstmc/server/CloudServer.java

```diff
@@ -1827,11 +1832,11 @@ public StorageRegistry getStorageRegistry() {
         return storageRegistry;
     }
 
-    public CloudGameRuleRegistry getGameRuleRegistry() {
+    public GameRuleRegistry getGameRuleRegistry() {
```

Comment:
These intentionally used the `Cloud` derivative to make it easier for people who want to use internal APIs.

Reply:
They can cast if they want, can't they? If we provide the internal implementation here we have to remove it from the Server interface [here](https://github.com/CloudburstMC/API/pull/13/commits/7eeb4f550d77d7f87a365efb9c79bc6af885d8c1#diff-9a226795261e9f581c3a23652f65343e6c5df40a47ce0691ce09924850904a6eR40)

I can remove it if you want me to.

Reply:
OH actually we don't need to remove from the interface sorry lol.

Reply:
Maybe we can make others return the implementation too? What do you think about that. 
I mean ItemRegistry and RecipeRegistry.